test/widgets/inbox_test.dart

```diff
@@ -383,6 +452,13 @@ void main() {
           checkAppearsCollapsed(tester, 1, findSectionContent);
           await tapCollapseIcon(tester, 1);
           checkAppearsUncollapsed(tester, 1, findSectionContent);
+
+          // check if collapsing stream through sticky header behaves
+          // as expected after scroll
+          await checkCollapseAfterScroll(tester, 2, true);
+          // check if collapsing stream through normal header behaves
+          // as expected after scroll
+          await checkCollapseAfterScroll(tester, 3, false);
```

Comment:
This should go in a new test, not as additional checks at the end of an existing test.

That's important for two reasons:
* It means that if some change breaks the other functionality that this test checks, or breaks this functionality, or breaks both of them, the test results clearly indicate which of those scenarios it is.
* It makes it possible to read and understand this test without first thinking through all the unrelated manipulation above it which is there for the sake of the existing test.